ostree-backup.sh

```diff
@@ -58,19 +58,14 @@ if [[ ! -f /tmp/recert_expired_certs.done ]]; then
     ETCD_IMAGE=$(jq -r '.spec.containers[] | select(.name == "etcd") | .image' </etc/kubernetes/manifests/etcd-pod.yaml)
     RECERT_IMAGE="quay.io/edge-infrastructure/recert:latest"
 
-    for container in recert_etcd recert; do
```

Comment:
Let's also fix this:

https://github.com/eranco74/sno-relocation-poc/blob/70198e8a2ea6db0367fbc701ee1521cd516e1d95/bake/installation-configuration.sh#L68-L70